cirq/ops/moment_test.py

```diff
@@ -261,3 +261,11 @@ def test_immutable_moment():
         circuit = cirq.Circuit(cirq.X(q1))
         moment = circuit.moments[0]
         moment.operations += (cirq.Y(q2),)
+
+
+def test_add():
```

Comment:
Add a test that works for a non-list circuit  (e.g) `circuit3 = cirq.Circuit(cirq.CNOT(a, b), cirq.X(a))`

Reply:
Done.